com.synology.SynologyDrive.yaml

```diff
@@ -12,6 +12,8 @@ finish-args:
   - --share=ipc
   - --talk-name=org.kde.StatusNotifierWatcher
   - --talk-name=org.freedesktop.Notifications
+  # Need for system tray icon to work
+  - --socket=session-bus
```

Comment:
Please use a specific dbus name instead.

Reply:
You are probably looking for `--own-name=org.kde.StatusNotifierItem-7-1`

Reply:
Thanks, I fixed it.